sys/fs/fuse/fuse_vnops.c

```diff
@@ -444,7 +446,8 @@ fuse_vnop_access(struct vop_access_args *ap)
 				return 0;
 			}
 		}
-		return EBADF;
+		return (EXTERROR(EBADF, "Access denied until FUSE session "
```

Comment:
OK, though FWIW I think this line is unreachable.  It's hard to prove that, though.